Client/mods/deathmatch/logic/rpc/CObjectRPCs.cpp

```diff
@@ -126,3 +127,11 @@ void CObjectRPCs::SetObjectBreakable(CClientEntity* pSource, NetBitStreamInterfa
         pObject->SetBreakable(bitStream.ReadBit());
     }
 }
+
+void CObjectRPCs::BreakObject(CClientEntity* pSource, NetBitStreamInterface& bitStream)
+{
+    CDeathmatchObject* pObject = static_cast<CDeathmatchObject*>(m_pObjectManager->Get(pSource->GetID()));
```

Comment:
you could use `auto` here to somehow increase readability.
Its clear that the output type is going to be `CDeathmatchObject*`